src/Middleware/CasProxyTicketSessionMiddleware.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace OpenEuropa\EPoetry\Middleware;
+
+use Http\Promise\Promise;
+use Phpro\SoapClient\Middleware\Middleware;
+use Phpro\SoapClient\Middleware\MiddlewareInterface;
+use Psr\Http\Message\RequestInterface;
+use Symfony\Component\HttpFoundation\Session\Session;
+
+class CasProxyTicketSessionMiddleware extends Middleware implements MiddlewareInterface
+{
+    private const PGT_ATTRIBUTE = 'cas_pgt';
+
+    /**
+     * The user session.
+     *
+     * @var Symfony\Component\HttpFoundation\Session\Session
+     */
+    protected $session;
+
+    /**
+     * CasProxyTicketMiddleware constructor.
+     *
+     * @param session $session
+     *   The user session
+     */
+    public function __construct(Session $session)
+    {
+        $this->session = $session;
+    }
+
+    public function beforeRequest(callable $handler, RequestInterface $request): Promise
```

Comment:
`{@inheritdoc}` is missing, I wonder why our CS components did not catch that.

Reply:
Fixed.